1_fetch/src/get_nwis_data.R

```diff
@@ -21,14 +21,19 @@ download_nwis_data <- function(filepath, parameterCd = '00010',
   return(filepath)
 }
 
+
+## Combine the site csv's
+combine_sites <- function(site_csv, file_out){
```

Comment:
Might be clearer to rename the argument to make it clear it is a vector of filepaths, not just for one site. E.g. `site_csvs`, `site_files` etc etc